css/properties/counter-reset.json

```diff
@@ -48,6 +48,56 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "reversed": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counter-reset",
```

Comment:
Since there's no standalone page for this feature and it's the same URL as the parent feature, omit the MDN URL.

```suggestion
```